project-base/src/Controller/Front/CartController.php

```diff
@@ -332,4 +339,34 @@ public function deleteAction(Request $request, $cartItemId)
 
         return $this->redirectToRoute('front_cart');
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $cartItemId
+     */
+    public function deleteAjaxAction(Request $request, $cartItemId)
+    {
+        $cartItemId = (int)$cartItemId;
+        $token = new CsrfToken('front_cart_delete_' . $cartItemId, $request->query->get('_token'));
+
+        if ($this->get('security.csrf.token_manager')->isTokenValid($token)) {
+            try {
+                $this->cartFacade->deleteCartItem($cartItemId);
+            } catch (\Shopsys\FrameworkBundle\Model\Cart\Exception\InvalidCartItemException $ex) {
+                return $this->json([
+                    'success' => false,
+                    'errorMessage' => $ex->getMessage(),
+                ]);
+            }
+        } else {
+            return $this->json([
+                'success' => false,
+                'errorMessage' => t('Unable to remove item from cart. The link for removing it probably expired, try it again.'),
```

Comment:
When I changed token to some invalid value, this error was not shown to me, I saw a blank modal window instead. 
Can you take a look at it?

<img width="505" alt="Snímek obrazovky 2020-01-19 v 22 05 53" src="https://user-images.githubusercontent.com/1177414/72688521-61836680-3b08-11ea-8dc2-7361294eb85f.png">

Reply:
Fixed, thx for report it